scss/components/_activity-big.scss

```diff
@@ -0,0 +1,123 @@
+.civicase__activity-card.civicase__activity-card--big {
```

Comment:
All this styles should be part of `activity-card.scss`, because this is still the same component, just a new variant. 
If you really want to create new file, then naming convention is wrong. Should be `activity-card--big.scss`

Reply:
Lets rename `_activity-big` to `_activity-card--big.scss`. That will be a better option.